Library/Formula/rabbitmq.rb

```diff
@@ -1,14 +1,10 @@
-# We'd really like this formula to be built from source so we can build
-# against a modern OpenSSL rather than the prehistoric system version.
-# Submit a PR if you're interested, Thanks!
 class Rabbitmq < Formula
   desc "Messaging broker"
   homepage "https://www.rabbitmq.com"
-  url "https://www.rabbitmq.com/releases/rabbitmq-server/v3.5.7/rabbitmq-server-mac-standalone-3.5.7.tar.gz"
-  sha256 "48a2acb30c1731f82303ef8fe3447a5f643ea7327bb8503e44149790ed7d5a7d"
-
-  bottle :unneeded
+  url "http://www.rabbitmq.com/releases/rabbitmq-server/v3.5.7/rabbitmq-server-generic-unix-3.5.7.tar.gz"
```

Comment:
By the way, we *could* also use a GitHub-hosted URL: https://github.com/rabbitmq/rabbitmq-server/releases/download/rabbitmq_v3_5_7/rabbitmq-server-generic-unix-3.5.7.tar.gz. Any reason to prefer one over the other?